ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/qos_profiles.schema.yml

```diff
@@ -154,3 +154,99 @@ keys:
                 type: str
                 description: |
                   Text comment added to queue.
+
+        priority_flow_control:
+          type: dict
+          description: |
+            Priority Flow Control settings
+          keys:
+            enabled:
+              type: bool
+              description: |
+                Enable Priority Flow control.
+            watchdog:
+              type: dict
+              description: |
+                Watchdog can detect stuck transmit queues.
+              keys:
+                enabled:
+                  type: bool
+                  required: true
+                  description: |
+                    Enable the watchdog on stuck transmit queues.
+                action:
+                  type: str
+                  valid_values: ["drop", "notify-only"]
+                  description: |
+                    Override the default error-disable action to either drop
+                    traffic on the stuck queue or notify-only
+                    without making any actions on the stuck queue.
+                timer:
+                  type: dict
+                  description: |
+                    Timer thresholds whilst monitoring queues.
+                  keys:
+                    timeout:
+                      type: str
+                      required: true
+                      pattern: "\\d+(\\.\\d)?"
+                      convert_types:
+                        - int
+                        - float
+                      description: |
+                        Timeout in seconds after which port should be errdisabled or
+                        should start dropping on congested priorities.
+                        This should be decimal with up to 1 decimal point
```

Comment:
apparently EOS prints this out as a two-decimal number. Please investigate and correct also for the global context.